docs/content/how-to/aws/troubleshooting.md

```diff
@@ -121,4 +121,26 @@ oc delete aws-ebs-csi-driver-controller aws-ebs-csi-driver-operator
 
 The operator will take a while to raise up again and eventually the driver controller will be deployed by the `aws-ebs-csi-driver-operator`.
 
-**Cause:** This issue probably come from objects that are deployed by the Operator, in this case the storage one, but the controller or the operator does not reconcile over them. If you delete the deployments you ensure that the operator recreates them from scratch.
\ No newline at end of file
+**Cause:** This issue probably come from objects that are deployed by the Operator, in this case the storage one, but the controller or the operator does not reconcile over them. If you delete the deployments you ensure that the operator recreates them from scratch.
+
+
+### The image-registry ClusterOperator keeps reporting a degraded status
+
+When a migration it's done and the image-registry clusteroperator it's marked as degraded, we need to figured it how it reaches to that status. The message will looks like `ImagePrunerDegraded: Job has reached the specified backoff limit`.
+
+Things we need to review:
+
+- Look for failure pods into the HostedControlPlane namespace at the Destination management cluster.
+- Check the other Cluster operators in the HostedCluster.
+- Check the Nodes if they are Ready and working fine in the HostedCluster.
+
+If all these three components are working fine, the issues relies in the backoff times the Job `image-pruner-XXXX` has been executed and has failed. Once the migrated cluster has already converged and looks fine, we need to make sure fix this cluster operator manually if we need inmediate resolution or just wait 24h and the cronjob will raise up another job by itself.
```

Comment:
```suggestion
If all three components are working fine, the issue is in the backoff times of the executed job `image-pruner-XXXX`; this job has most likely failed. Once the migrated cluster has already converged and looks fine, you will need to make sure to fix this cluster operator manually; you will need to determine if you want immediate resolution, or you can wait 24h and the cronjob will raise up another job by itself.
```

Reply:
Some of your original sentences didn't make sense to me; so, I tried to reword some of it to capture what I thought you were saying. Feel free to trash what I did especially if I didn't capture the right meaning :)